src/pages/AddDoctor.js

```diff
@@ -0,0 +1,6 @@
+function AddDoctor() {
+  return (
+    <h2>This is AddDoctor page</h2>
+  );
+}
+export default AddDoctor;
```

Comment:
Hey @mahdinoori2000 thank you for the additions and the wonderful work done.
Could you please try to use arrow functional components for these pages as they seem to be better and cleaner code.